docs/ops.html

```diff
@@ -477,16 +477,20 @@ <h4><a id="rep-throttle" href="#rep-throttle">Limiting Bandwidth Usage during Da
   Throttle was removed.</pre>
 
   <p>The administrator can also validate the assigned configs using the kafka-configs.sh. There are two pairs of throttle
-      configuration used to manage the throttling process. The throttle value itself. This is configured, at a broker
+      configuration used to manage the throttling process. This is configured, at a broker
       level, using the dynamic properties: </p>
 
-  <pre class="brush: text;">leader.replication.throttled.rate
-  follower.replication.throttled.rate</pre>
+  <pre class="brush: text;">
+    leader.replication.throttled.rate
+    follower.replication.throttled.rate
+  </pre>
 
   <p>There is also an enumerated set of throttled replicas: </p>
```

Comment:
Then here you could amend the sentence to say:
```suggestion
  <p>Then there is the configuration pair of throttled replicas: </p>
```

Reply:
Good suggestion! after the update, it's more clear!
![image](https://user-images.githubusercontent.com/43372967/82292384-6506fb80-99dd-11ea-8e99-15ed86db1186.png)